models/delivery.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var deliveryModel = require('hmpo-model');
```

Comment:
Aah, I was viewing it as more traditional inheritance vs partials... I'll
read up some

Thanks

On Fri, 19 Jun 2015, 21:57 Joe Chapman <notifications@github.com> wrote:

> In models/delivery.js
> <https://github.com/UKHomeOffice/brp_app/pull/6#discussion_r32868722>:
>
> > @@ -0,0 +1,32 @@
> > +'use strict';
> > +
> > +var deliveryModel = require('hmpo-model');
>
> look at this: https://github.com/UKHomeOffice/passports-model
> when you add to the prototype of a constructor, you are essentially
> defining new methods on any new instance of that constructor/class. So if
> someone else new's up that constructor/class they'll share any new
> prototype methods you've added
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UKHomeOffice/brp_app/pull/6/files#r32868722>.
>